docs/manual-installation.rst

```diff
@@ -25,7 +23,30 @@ Ubuntu
         - ``cd ~/harvest/alcazard``
         - ``virtualenv -p python3 venv``
 4. Install dependencies
-    1. ``cd ~/harvest``
-    2. ``pip3 install -r requirements.txt``
-    3. ``cd ~/harvest/alcazard``
-    4. ``pip3 install -r requirements.txt``
+    1. Harvest dependencies
+        - ``cd ~/harvest``
+        - ``pip3 install -r requirements.txt``
+    2. Alcazard dependencies
+        - ``cd ~/harvest/alcazard``
+        - ``pip3 install -r requirements.txt``
+5. Install and configure PostgreSQL
+    1. Install the PostgreSQL packages
+        - ``sudo apt install postgresql postgresql-contrib``
+    2. Configure PostgreSQL to start up upon bootup then manually start it
+        - ``sudo update-rc.d postgresql enable``
+        - ``sudo service postgresql start``
+    3. Create a PostgreSQL user; this can be any desired user
+        - ``sudo -u postgres createuser USER``
+    4. Create the SQL database. These instructions use "harvest" as the db name
+        - ``sudo -u postgres createdb --owner USER harvest``
+6. Create a file named ``django_env`` in ~/harvest
+        - Paste these contents inside the file ``DJANGO_DB=postgres:///harvest``
+7. Migrate the database
+    - ``./manage.py migrate``
+8. Install nodesource
```

Comment:
Maybe "Install node.js from the official apt repos" and then split the npm install / npm run build into a separate step "Compile Harvest JS resources"